src/docs/get-started/flutter-for/web-devs.md

```diff
@@ -119,10 +119,18 @@ the top left, by default.
 
 ### Setting background color
 
+<<<<<<< HEAD
 ### 设置背景颜色
 
 In Flutter, you set the background color using a
 [`Container`][]’s `decoration` property.
+=======
+In Flutter, you set the background color using the 
+`color` property or the `decoration` property of a [`Container`][]. However, you cannot
+supply both, since it would potentially result in the decoration drawing over
+the background color. The `color` property should be preferred when the background is a simple color.
+For other cases, such as gradients or images, use the `decoration` property.
+>>>>>>> 2341c639fdfcf7aeef464a95dabab604bbf12532
```

Comment:
Here we have some unfinished merge process.

Reply:
fixed.